### DISCUSSION TITLE
Setting transparency

### DISCUSSION THREAD

**Discussion by Fuhrerbaumanki** [Avdushin/GOAPPLE#11] [Q&A] [answered]
Which file can be used to configure transparency settings? Let's say I want to make it so that Fire Fox Was transparent0,9 When active and 0.7 when inactive. Where can I set these parameters and which utility is responsible for transparency in general?

**Comment by Avdushin** [comment, marked as answer]
Greetings @Fuhrerbaumanki!  The [picom.conf](https://github.com/Avdushin/GOAPPLE/blob/main/src/distros/Manjaro/i3/picom.conf) file is responsible for window transparency.  In it you can adjust the rules of transparency, blur and other effects for windows...

For Example:
```bash
opacity-rule = [
  "80:class_g     = 'Bar'",             # lemonbar
  "100:class_g    = 'slop'",            # maim
  "100:class_g    = 'XTerm'",
  "100:class_g    = 'URxvt'",
  "100:class_g    = 'kitty'",
  "100:class_g    = 'Alacritty'",
  "80:class_g     = 'Polybar'",
  "100:class_g    = 'code-oss'",
  "100:class_g    = 'Meld'",
  "100:class_g     = 'TelegramDesktop'",
  "90:class_g     = 'Joplin'",
  "100:class_g    = 'firefox'",
  "100:class_g    = 'Thunderbird'",
  "100:class_g    = 'Google-chrome'",
  "100:class_g    = 'Polybar'",
  "99:class_g    = 'Thunar'",
  "100:class_g    = 'Sublime_text'",
  "100:class_g    = 'Steam'",
];
```
